src/GeoIp2/WebService/Client.php

```diff
@@ -62,13 +64,17 @@ class Client implements ProviderInterface
      * @param string $host Optional host parameter
      * @param object $guzzleClient Optional Guzzle client to use (to facilitate
      * unit testing).
+     * @param string $timeout Total transaction timeout
+     * @param string $connectTimeout Initial connection timeout
      */
     public function __construct(
         $userId,
         $licenseKey,
         $locales = array('en'),
         $host = 'geoip.maxmind.com',
-        $guzzleClient = null
+        $guzzleClient = null,
+        $timeout = null,
+        $connectTimeout = null
```

Comment:
I wish we would have used an options array here as this is getting a bit unmanageable, but I guess that is water under the bridge.